azure-sb.js

```diff
@@ -22,6 +22,7 @@ var azuresbServers = require(path.join(configDir, 'azure-sb.json')).map((c) => {
   client.prefix = c.prefix;
   client.queues = c.queues;
   client.topics = c.topics;
+  client.ignoredsubscriptions = c.ignoredsubscriptions;
```

Comment:
if ignoredsubscriptions isn't present in the config it'll be null and you'll get an exception when you try to access it, we need to set the value to an empty array if not present.

you can use a falsy-coalesce: `client.ignoredsubscriptions = c.ignoredsubscriptions || [];`